src/Tools/dotnet-stack/dotnet-stack.csproj

```diff
@@ -11,6 +11,9 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.DiaSymReader" Version="1.4.0-beta2-20217-02" />
+    <PackageReference Include="Microsoft.DiaSymReader.Converter.Xml" Version="1.1.0-beta2-20216-01" />
+    <PackageReference Include="Microsoft.DiaSymReader.PortablePdb" Version="1.6.0-beta2-21126-02"/>
```

Comment:
@hoyosjs are these packages managed under DARC or should we leave the manual versioning here?

Reply:
I'd centralize them. Also, we already use Dia in SOS / dotnet dump.